diff --git a/node_modules/react-native-reanimated/lib/module/hook/useAnimatedStyle.js b/node_modules/react-native-reanimated/lib/module/hook/useAnimatedStyle.js
index 1c246bf..2327eb6 100644
--- a/node_modules/react-native-reanimated/lib/module/hook/useAnimatedStyle.js
+++ b/node_modules/react-native-reanimated/lib/module/hook/useAnimatedStyle.js
@@ -349,9 +349,9 @@ For more, see the docs: \`https://docs.swmansion.com/react-native-reanimated/doc
         isAnimationRunning: false
       }),
       viewDescriptors: makeViewDescriptorsSet(),
-      styleUpdaterContainer: {
+      styleUpdaterContainer: Object.seal({
         current: undefined
-      }
+      })
     };
   }
   const {
diff --git a/node_modules/react-native-reanimated/src/hook/useAnimatedStyle.ts b/node_modules/react-native-reanimated/src/hook/useAnimatedStyle.ts
index 877d8d9..978bb5c 100644
--- a/node_modules/react-native-reanimated/src/hook/useAnimatedStyle.ts
+++ b/node_modules/react-native-reanimated/src/hook/useAnimatedStyle.ts
@@ -521,7 +521,7 @@ For more, see the docs: \`https://docs.swmansion.com/react-native-reanimated/doc
         isAnimationRunning: false,
       }),
       viewDescriptors: makeViewDescriptorsSet(),
-      styleUpdaterContainer: { current: undefined },
+      styleUpdaterContainer: Object.seal({ current: undefined }),
     };
   }
 
